samples/sqltransport/multi-instance/Client/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
```

Comment:
because all the info in an assembly info is about deployment of a library. you are never going to deploy a sample so it serves no purpose